src/tests/Sort.test.js

```diff
@@ -0,0 +1,41 @@
+const { app } = require('../pages/Application'); 
+
+describe('Inventory Sorting', () => {
+    before(async () => {
+        await app.login.navigate();
+        await app.login.performLogin();
+    });
+
+    it('should sort items by name (Z to A)', async () => {
+        await app.inventory.navigate();
+
+        const inventoryItems = await app.inventory.inventoryItems;
+        expect(inventoryItems.length).toBeGreaterThan(0);
+
+        const initialNames = [];
+        for (const item of inventoryItems) {
+            const itemName = await app.inventory.getItemName(item).getText();
+            initialNames.push(itemName);
+        }
+
+        console.log('Initial names:', initialNames);
+
+        const sortDropdown = await app.inventory.sortDropdown;
+        expect(await sortDropdown.isDisplayed()).toBe(true);
+        await sortDropdown.selectByAttribute('value', 'za');
+
+        const updatedInventoryItems = await app.inventory.inventoryItems;
+        const sortedNames = [];
+        for (const item of updatedInventoryItems) {
+            const itemName = await app.inventory.getItemName(item).getText();
+            sortedNames.push(itemName);
+        }
+
+        console.log('Sorted names:', sortedNames);
+
+        const expectedSortedNames = [...initialNames].sort().reverse();
+        console.log('Expected Sorted Names:', expectedSortedNames);
+
+        expect(sortedNames).toEqual(expectedSortedNames);
+    });
```

Comment:
I took your code and modified it a little bit to show how the solution can be improved. Provided code does the same. So, the test might look like:

<img width="1370" alt="image" src="https://github.com/user-attachments/assets/52bb243d-57f0-4fe0-946d-26be31e729b0">

Unused method was deleted from the pageObject  and two new methods were added (they are used in the test)

<img width="1394" alt="image" src="https://github.com/user-attachments/assets/ebb627b2-1432-4b27-b490-2b6071272b1f">

Also, Two files with helpers were added. The first one for sorting methods to put the sorting logic aside and not to keep it in the each test (It does not relate to the bageObject itself, so I decided to move them). In the future you might want to sort other arrays by names/numbers(in our case prices, so I named it like that)

<img width="1394" alt="image" src="https://github.com/user-attachments/assets/e74aa860-8b44-4f41-9e60-e59cf52a3a35">

The second helper is for the sorting options to avoid 'magic strings' and typos

<img width="1394" alt="image" src="https://github.com/user-attachments/assets/4c360716-d38b-4e22-b96d-7cb79433c280">

This is only one way of many to improve. So, I suggest trying to implement the same solution just for practice. Let me know if you need src